tests/modules/programs/scmpuff/fish.nix

```
@@ -0,0 +1,21 @@
+{ pkgs, lib, ... }: {
+  config = {
+    programs = {
+      scmpuff.enable = true;
+      fish.enable = true;
+    };
+
+    # Needed to avoid error with dummy fish package.
+    xdg.dataFile."fish/home-manager_generated_completions".source =
+      lib.mkForce (builtins.toFile "empty" "");
+
+    test.stubs.fish = { };
+
+    nmt.script = ''
+      assertFileExists home-files/.config/fish/config.fish
+      assertFileContains \
+        home-files/.config/fish/config.fish \
+        '${pkgs.scmpuff}/bin/scmpuff init -s --shell=fish | source'
```

```suggestion
        '@scmpuff@/bin/scmpuff init -s --shell=fish | source'
```
If the package is overriden with `test.stubs`, its `outPath` will be changed to `@scmpuff@` by default.

---

Done